packages/ai-jsx/src/batteries/constrained-output.tsx

```diff
@@ -78,9 +79,18 @@ export type TypedObjectCompletionWithRetry = TypedObjectCompletion & { retries?:
  *    Intermediate results that are valid, are also yielded.
  */
 export async function* JsonChatCompletion(
-  props: Omit<TypedObjectCompletionWithRetry, 'typeName' | 'parser' | 'partialResultCleaner'>,
+  { schema, ...props }: Omit<TypedObjectCompletionWithRetry, 'typeName' | 'parser' | 'partialResultCleaner'>,
   { render }: AI.ComponentContext
 ) {
+  if (schema) {
```

Comment:
I think it would actually be nice to always use `JsonChatCompletionFunctionCall`. If no `schema` is provided, that means the user just wants JSON of any form, right? In that case, we can create a very generic Zod schema that matches any JSON (maybe just `zod.object()`? 

https://www.perplexity.ai/search/56487aec-baf4-4590-bd66-c43a713b0127?s=c

Reply:
Good idea. Done.